src/inet/physicallayer/ieee80211/packetlevel/Ieee80211ReceiverBase.cc

```diff
@@ -81,6 +81,7 @@ void Ieee80211ReceiverBase::setChannel(const Ieee80211Channel *channel)
         delete this->channel;
         this->channel = channel;
         setCenterFrequency(channel->getCenterFrequency());
+        setBandwidth(channel->getBand()->getSpacing());
```

Comment:
Band spacing is not the same as the listening bandwidth, so I think this is wrong.

Reply:
It is true, but the channel structure only has information about the spacing and center, not the bandwidth, I have tried to include the minimum changes, without this modification it is impossible to use HT and VHT modes with bandwidths bigger than 20MHz. It should be necessary to include information about the bandwidth in the IIeee80211Band class. If you are busy, I can do it.

Reply:
Indeed, bandwidth should added to that interface and subclasses. You can give it a try, but please create a new pull request. This is unrelated to the others and it would be better to discuss and merge them separately.

Reply:
Ok